insitupy/campaigns/campaign.py

```diff
@@ -37,8 +34,6 @@ class ProfileDataCollection:
     """
     META_PARSER = MetaDataParser
     PROFILE_DATA_CLASS = ProfileData
-    DEFAULT_METADATA_VARIABLE_FILES = [base_metadata_variables_yaml]
-    DEFAULT_PRIMARY_VARIABLE_FILES = [base_primary_variables_yaml]
 
```

Comment:
This was the actual bug I was looking for. We need to delegate this definition to the `PROFILE_DATA_CLASS` so the inheritance for SnowExCollections would work.